addon/components/new-osf-navbar/style.scss

```diff
@@ -0,0 +1,228 @@
+// Overrides some of base osf-styles for the navbar in osf-style/sass/_bootstrap.scss
+$osf-teal-navbar: #2E6A74;
+$osf-highlight-navbar: #5A8288;
+$osf-dark-blue-navbar: #263947;
+
+.new-osf-navbar-styling {
+    /* Overrides the overall navbar color */
+    .navbar-inverse {
+        background-color: $osf-dark-blue-navbar;
+    }
+
+    /* Enlarges OSF Font */
+    .navbar-brand {
+        font-size: 28px;
+    }
+
+    /* Enlarges OSF logo */
+    .osf-navbar-logo {
+        width: 35px;
+        height: 35px;
+        margin-top: -4px;
+        margin-left: -4px;
+    }
+
+    /* Defines position of primary navigation: dropdown toggle (caret) */
+    .primary-nav {
+        float: left;
+        /* Overrides vertical and horizontal position of dropdown caret for service menu */
+        li.dropdown {
+            margin-left: -46px;
+            padding-top: 15px;
+            list-style-type: none;
+            font-size: 15px;
+        }
+    }
+
+    /* Overrides the color, relative position, and font size of dropdown menus (both service and auth) */
+    .dropdown-menu {
+        background-color: $osf-teal-navbar;
+        &.service-dropdown {
+            li > a {
+               font-size: 16px;
+            }
+            top: 58px;
+            left: -14.5px;
+        }
+        &.auth-dropdown {
+            li > a {
+               font-size: 15px;
+            }
+            top: 60px;
+            left: 15px;
+        }
+    }
+
+    /* Overrides background of dropdown-toggle when active - needed for auth navbar */
+     .navbar-nav > .open > a,
```

Comment:
Will this navbar be used in the regular OSF too?

Currently, navbar CSS lives in the [osf-style](https://github.com/CenterForOpenScience/osf-style) repo. For the sake of maintainability, I think we should consider putting the styles for this shared product in that shared location. Then we would maintain a single location for visual elements of both the flask and ember apps.